config.go

```diff
@@ -281,6 +287,8 @@ type Config struct {
 	MinBackoff        time.Duration `long:"minbackoff" description:"Shortest backoff when reconnecting to persistent peers. Valid time units are {s, m, h}."`
 	MaxBackoff        time.Duration `long:"maxbackoff" description:"Longest backoff when reconnecting to persistent peers. Valid time units are {s, m, h}."`
 	ConnectionTimeout time.Duration `long:"connectiontimeout" description:"The timeout value for network connections. Valid time units are {ms, s, m, h}."`
+	SOCKS             string        `long:"socks" description:"The host:port that SOCKS5 proxy to use for any outbound connections not going over Tor"`
```

Comment:
Weird sentence. Did you mean to say `The host:port of the SOCKS5 proxy to use for any...`?